Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -15,13 +14,13 @@ public abstract class BaseWorldManager : MonoBehaviour
         [Title("Base Component")]
         [SerializeField] protected PlayerBehaviour _playerBehaviour = null;
 
-        private IInputSettings _inputSettings = null;
-        private FreezeActionsUtility _freezeActionsUtility = null;
-
-        private EnvironmentManager _environmentManager = null;
+        [SerializeField] private InputActionAsset _actionAsset;
```

Comment:
`InputActionAsset` is used for `PGInputSystem`. 
`PGInputSystem` handles all the 3rd party - `new InputSystem`, and can be passed into whatever needs it. e.g. ActionMap mapping (swapping between UI And player control)